src/bust.py

```diff
@@ -314,7 +316,18 @@ def get_google_form_url(self, image_url: Optional[str] = None) -> Optional[str]:
     async def send_stats(self, interaction: Interaction) -> None:
         songs_len = int(self.total_song_len)
         num_songs = len(self.bust_content)
-        bust_len = songs_len + config.seconds_between_songs * num_songs
+        bust_len_in_seconds = songs_len + config.seconds_between_songs * num_songs
```

Comment:
instead of calculating both `bust_len_in_seconds` and `remaining_bust_len_in_seconds` and just showing one based on the if statement, you should probably just have one variable which branches based on whether we're busting or not